### DISCUSSION TITLE
change pool to an exists pool

### DISCUSSION THREAD

**Discussion by mcjmcj** [Chia-Network/chia-blockchain#7821] [Ideas]
if I have 2 pools and I want to change one  this to the another then will be create a new lancher id, my idea is change one pool to the another exists pool launcher id

**Comment by keliew** [comment]
1 Launcher ID (NFT) per pool.

https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide

If you want to be in 2 pools with separate plots, then you'd need 2 NFTs of plots.

**Comment by mcjmcj** [comment]
Launcher ID is not unique per pool. If I have a pool for example Space Pool and XCH Pool and want to migrate XCH to Space then it creates a second launcher ID, I would like to be able to play XCH pool to the same snackr ID as my current Space pool

________________________________
De: keliew ***@***.***>
Enviado: quarta-feira, 28 de julho de 2021 04:32
Para: Chia-Network/chia-blockchain ***@***.***>
Cc: mcjmcj ***@***.***>; Author ***@***.***>
Assunto: Re: [Chia-Network/chia-blockchain] change pool to an exists pool (#7821)


1 Launcher ID (NFT) per pool.

https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide

If you want to be in 2 pools with separate plots, then you'd need 2 NFTs of plots.

**Reply by keliew** [reply]
You don't need another NFT to change pools...
https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide#step-5-manage-your-plot-nft

**Comment by mcjmcj** [comment]
I think I don't know how to explain myself.


I have 2 lancher ID, 1 of SPACE pool and another of XCH pool.

I want change plots of XCH pool to SPACE pool, but if I do this, will create the new lancher ID of SPACE pool. I want migrate XCH pool to the exists lancher ID of  SPACE pool.

________________________________
De: keliew ***@***.***>
Enviado: quarta-feira, 28 de julho de 2021 10:09
Para: Chia-Network/chia-blockchain ***@***.***>
Cc: mcjmcj ***@***.***>; Author ***@***.***>
Assunto: Re: [Chia-Network/chia-blockchain] change pool to an exists pool (#7821)


You don't need another NFT to change pools...
https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide#step-5-manage-your-plot-nft

**Reply by keliew** [reply]
It's not possible to migrate plots to another NFT (Launcher ID); it is baked/infused into the plots. That's how Chia can identify which plots can farm to which pool, even when you put everything in the same folder.

https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-FAQ#what-is-a-plot-nft
